js/apps/admin-ui/src/user/UsersSection.tsx

```diff
@@ -104,10 +158,25 @@ export default function UsersSection() {
         storageProviders.filter((p) => p.config?.enabled[0] === "true")
       );
       setRealm(realm);
+      fetchUserProfileAttribute(realm);
     },
     []
   );
 
+  const isBundleKey = (displayName?: string) => displayName?.includes("${");
+  const unWrap = (key: string) => key.substring(2, key.length - 1);
+
+  const fetchUserProfileAttribute = (
+    realm: RealmRepresentation | undefined
+  ) => {
+    if (realm?.attributes?.userProfileEnabled === "true") {
+      setUserProfileEnabled(true);
+      adminClient.users.getProfile().then((profile) => {
```

Comment:
It's possible to call this API, even when this feature is not enabled. Since it has no real dependencies I think it might be better for performance to just make this call regardless.

This also means that this API call can be made directly from the same place the other API calls are being made in `useFetch()`.

Reply:
Yea right, got it, thank you. 👍 I made the necessary changes and tried to simplify the code overall. You are very welcome to take another look at the whole file. 🙏